modules/react/switch/lib/Switch.tsx

```diff
@@ -92,15 +92,15 @@ const switchInputStencil = createStencil({
       error: {
         '& ~ div:first-of-type': {
           boxShadow: `
-              0 0 0 ${px2rem(2)} ${base.frenchVanilla100},
+              0 0 0 ${px2rem(2)} ${system.color.fg.inverse},
```

Comment:
Should this be `system.color.bg.default` ?

Reply:
It should be `fg` for box shadows